### DISCUSSION TITLE
Any way to switch theme/iconTheme/cat from external program/script?

### DISCUSSION THREAD

**Discussion by MidyGamy** [PrismLauncher/PrismLauncher#3124] [Q&A]
I there any way to switch the theme, icon theme or cat from an external program or bash script?

I currently use the breeze icons to go with my KDE plasma DE on KDE Neon and I'm using [Koi](https://github.com/baduhai/Koi) to automatically switch between light and dark theme... I works for the global Prism Launcher interface, but since the icon theme is managed by the launcher and not KDE, well I wanted to know if that was possible.

![image](https://github.com/user-attachments/assets/15741ac7-d45e-43a4-bf72-79ea7a521df1)
![image](https://github.com/user-attachments/assets/e14e9652-c520-4b69-883f-d41b07b94cff)

**Comment by Trial97** [comment]
Here is a suggestion:
- Stop prism (`pkill prismlauncher`)
- Using a program to change `prismlauncher.cfg`(`awk`/`sed` - there may be more) here are the config that controls all that
   - "IconTheme" - icons
   - "ApplicationTheme" - application
   - "BackgroundCat" - the cat
   - "TheCat" - if the cat is visible  
- Start prism (`prismlauncher&`)

Right now, there is no way to do all of this directly using command arguments in Prism, and there is also no way to get the possible values for the mentioned values.
The Prism restart mentioned is so the settings to be reread from file and for the theme change
